core/src/test/java/org/hibernate/ogm/backendtck/inheritance/singletable/family/SingleTableInheritancePersistTest.java

```diff
@@ -45,7 +45,9 @@ public void setUp() {
 	@After
 	public void tearDown() {
 		try {
-			em.getTransaction().begin();
+			if ( !em.getTransaction().isActive() ) {
+				em.getTransaction().begin();
+			}
```

Comment:
Wondering if we should fix that or the test itself as it's not a good example of how things should be done.

But maybe we have all our tests like that and it's not worth fixing.

Reply:
I think it might be because of the way the test is written, but I will check before sending a clean PR

Reply:
It seems this change is not actually necessary, I probably  got tricked by some other exceptions I hadn't solved yet